examples/cdk/lib/example-function.MyFunction.ts

```diff
@@ -38,24 +43,30 @@ export const handler = async (_event: unknown, context: Context): Promise<void>
   metricWithItsOwnDimensions.addMetric('single-metric', MetricUnits.Percent, 50);
 
   metrics.publishStoredMetrics();
-  metrics.raiseOnEmptyMetrics();
-
-  // ### Experiment tracer
+  metrics.throwOnEmptyMetrics();
 
-  tracer.putAnnotation('Myannotation', 'My annotation\'s value');
+  // ### Experiment with Tracer
+  // This annotation & metadata will be added to the handlerSegment subsegment (## index.handler)
+  tracer.putAnnotation('awsRequestId', context.awsRequestId);
+  tracer.putMetadata('eventPayload', event);
 
-  // Create subsegment & set it as active
-  const subsegment = handlerSegment.addNewSubsegment('MySubSegment');
+  // Create another subsegment & set it as active
+  const subsegment = handlerSegment.addNewSubsegment('### MySubSegment');
+  tracer.setSegment(subsegment);
 
+  let res;
   try {
-    throw new Error('test');
-    // Add the response as metadata
+    res = { foo: 'bar' };
+    tracer.addResponseAsMetadata(res, process.env._HANDLER);
   } catch (err) {
     // Add the error as metadata
     subsegment.addError(err as Error, false);
```

Comment:
Why do we need try/catch here if there is no throw?

Reply:
I know in this specific example there's no way that an exception will be thrown in the `try` block, I still added the whole `try/catch/finally` to give an example of the whole function and also to demonstrate how customers can add annotations in the `catch` & close the subsegments in the `finally`.
To sum up, I was intending these examples as a way for customers to copy/paste them and get started writing their own function.

Reply:
I just re-read the comment, which I misunderstood, sorry, you're right. Adding it!

Reply:
Added in [9c2a418](https://github.com/awslabs/aws-lambda-powertools-typescript/pull/439/commits/9c2a4189f9cfc22a24b10e10d68ba2b4108d2117)

Reply:
Discussed this offline and comments have been addressed.